site/docs/main/resource-filtering.md

```diff
@@ -0,0 +1,129 @@
+# Resource Filtering
```

Comment:
This file needs to be copied to `site/docs/v1.4` as well.

Reply:
Added.